# Interval of loop counts to publish ROS message
publish_loop_interval: 10

# Number of IK loop
ik_loop_num: 100

# Threshold of IK [m], [rad]
ik_error_thre: 5e-3

# Body name list for collision avoidance
collision_body_names_list:
  ["Lleg_Link2", "Rleg_Link2",
  "Lleg_Link3", "Rleg_Link3",
  "Lleg_Link5", "Rleg_Link5"]

# Upper limit of footstep sampling position [m], [deg]
upper_footstep_pos: [0.5, 0.6, 60]

# Lower limit of footstep sampling position [m], [deg]
lower_footstep_pos: [-0.5, -0.1, -60]

# Waist height [m]
waist_height: 0.7
